README.md

```diff
@@ -1,2 +1,9 @@
 # miranum-consulting
-This repository holds examples which are used and maintained by the consulting team for miranum  
+This repository holds examples which are used and maintained by the consulting team for miranum.  
+The following ones are contained in this repository: 
+* [Pizza-order Showcase](./pizza-order-showcase) 
+* [Gateway example](./gateway-example) 
+* [Schema Client example](./schema-client-example) 
+* [Process Example](./process-example) 
+
+More information can be found in [our documentation](https://miranum.com). 
```

Comment:
Good catch - I removed surfer but still included Lombok and maven.

Reply:
I would keep it like this. Because if you click on one you will be directed to the module. GitHub shows the readme (if existent) right away. (I would not optimise it for an IDE)